api/v1alpha1/imageclusterinstall_types.go

```diff
@@ -61,10 +83,17 @@ type ClusterConfigSpec struct {
 	BareMetalHostRef *BareMetalHostReference `json:"bareMetalHostRef,omitempty"`
 }
 
-// ClusterConfigStatus defines the observed state of ClusterConfig
-type ClusterConfigStatus struct {
+// ImageClusterInstallStatus defines the observed state of ImageClusterInstall
+type ImageClusterInstallStatus struct {
+	// Conditions is a list of conditions associated with syncing to the cluster.
+	// +optional
+	Conditions []hivev1.ClusterInstallCondition `json:"conditions,omitempty"`
+
+	// InstallRestarts is the total count of container restarts on the clusters install job.
+	InstallRestarts int `json:"installRestarts,omitempty"`
+
 	BareMetalHostRef *BareMetalHostReference `json:"bareMetalHostRef,omitempty"`
-	Conditions       []metav1.Condition      `json:"conditions,omitempty"`
+	ConfigConditions []metav1.Condition      `json:"configConditions,omitempty"`
```

Comment:
@eranco74 any thoughts on the name here?

Reply:
Or if we should force these into the existing `Conditions` field.

Reply:
The only potential issue I saw with that is the way the hive `ClusterInstallCondition` was written.
It has specific types and I'm wondering if anything would have a problem with us not using one of the defined ones because I don't think our conditions fit in the existing ones.

I can give this a try though.

ref: https://github.com/openshift/hive/blob/de169ddf445c3e064cfa8773de5af8ec00f649be/apis/hive/v1/clusterinstall_conditions.go